tests/read_test.cpp

```diff
@@ -5,7 +5,7 @@ void readtest(const ReadCase &current_case)
 
   auto       read_test = aste::BaseName{current_case.fname}.with(aste::ExecutionContext());
   aste::Mesh mesh;
-  read_test.loadMesh(mesh, current_case.dim);
+  read_test.loadMesh(mesh, current_case.dim, true);
```

Comment:
The test is still failing. Now that we have the nice boost test infrastructure: can you add a test for both cases, i.e., `requireConnectivity=true` and `requireConnectivity=false`?

Reply:
Sure I will take care of it afternoon

Reply:
Now it is restructured